.github/workflows/functional-test-noncloud.yaml

```diff
@@ -80,6 +80,8 @@ env:
   LOCAL_REGISTRY_SERVER: "localhost"
   # Local Docker registry port
   LOCAL_REGISTRY_PORT: "5000"
+  # bicep-types ACR url for uploading Radius Bicep types
+  BICEP_TYPES_REGISTRY: 'biceptypes.azurecr.io'
```

Comment:
Shouldn't this point to the local registry?

Reply:
No, this is used to pull the AWS ACR types. anonymous pull is enabled since it's a publicly readable asset so it shouldn't require any secrets